packages/core/src/__tests__/factory.test.ts

```diff
@@ -0,0 +1,47 @@
+// @ts-ignore
+import logger from 'abstract-logging';
+import { right } from 'fp-ts/lib/Either';
+import { asks } from 'fp-ts/lib/ReaderEither';
+import { Logger } from 'pino';
+import { factory, IPrismConfig } from '..';
+
+describe('validation', () => {
+  const validator = {
+    validateInput: jest.fn().mockReturnValue(['something']),
```

Comment:
`jest.fn().mockReturnValue(['something'])` can be put in a variable and reused below

Reply:
I cannot do this — the returned mocked function has an internal counter that tracks the number of calls and they need to be separate.